### ISSUE TITLE
WalletConnect demo not working

### ISSUE THREAD

**Issue by miguelmota** [blocknative/web3-onboard#2019]
### Current Behavior

Click on WalletConect option doesn't show QR code and appears stuck. Error is displayed on console.

```js
index.051cf284.js:19 TypeError: Cannot convert undefined or null to object
    at Function.values (<anonymous>)
    at Object.getRecomendedWallets (index.051cf284.js:15:10360)
    at async wt.preloadListings (index.051cf284.js:19:52678)
    at async Promise.all (/index 0)
    at async wt.preloadData (index.051cf284.js:19:52988)
```

### Expected Behavior

QRCode to show when clicking on WalletConnect 

### Steps To Reproduce

1. Visit https://onboard.blocknative.com/
2. Click Connect
3. Click WalletConnect
4. Observe no QRCode shown and console error

![DeepinScreenshot_select-area_20231206182419](https://github.com/blocknative/web3-onboard/assets/168240/4903c909-2130-4b38-b6de-047e19bbb0cb)

### What package is effected by this issue?

@web3-onboard/walletconnect

### Is this a build or a runtime issue?

Runtime

### Package Version

latest

### What browsers are you seeing the problem on?

Chrome

### Sanity Check

- [X] If this is a build issue, I have included my build config. If this is a runtime issue, I have included reproduction steps and/or a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example).

**Comment by Adamj1232** [comment]
@miguelmota thanks for raising this issue! Im in contact with the WalletConnect team to understand and resolve this issue

**Comment by Adamj1232** [comment]
The docs site has been updated and there is an alpha version of the walletconnect module that contains a fix for this which is available at `v2.5.2-alpha.1` and will be officially released shortly
